torch/csrc/jit/source_range.cpp

```diff
@@ -42,6 +42,8 @@ C10_EXPORT void SourceRange::highlight(std::ostream& out) const {
   }
   AT_ASSERT(end_highlight == str.size() || str[end_highlight] == '\n');
 
+  out << "at " << file_->filename_ << ":"
+      << file_->lineno_for_offset(start()) + file_->file_offset_ << "\n";
```

Comment:
Can we do what clang/gcc do and have `<lineno>:<column>` instead of only line number? Might be useful for machine parsers (for example, vim regexes on that pattern to generate a quickfix list).